### ISSUE TITLE
hostname incompatibilty

### ISSUE THREAD

**Issue by hoont** [AquaQAnalytics/TorQ#317]
`torq.sh` fails on arch-based systems due to an incompatibility with the version of `hostname` being used - see https://github.com/sorki/advanced-shell-history/issues/12.

```
$ ./torq.sh start all 

hostname: invalid option -- 'I'
Try 'hostname --help' or 'hostname --usage' for more information.
hostname: invalid option -- 'A'
Try 'hostname --help' or 'hostname --usage' for more information.
```
```
$ hostname --usage

Usage: hostname [-adfisy?V] [-F FILE] [--aliases] [--domain] [--fqdn] [--long]
            [--file=FILE] [--ip-addresses] [--short] [--yp] [--nis] [--help]
            [--usage] [--version] [NAME]
```
```
$ hostname -v

hostname (GNU inetutils) 1.9.4
Copyright (C) 2015 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```
```
DISTRIB_ID=ManjaroLinux
DISTRIB_RELEASE=20.0.3
DISTRIB_CODENAME=Lysia
DISTRIB_DESCRIPTION="Manjaro Linux"
```

**Comment by andrew-west** [comment]
Hi,

Can you try using the `torq.sh` file from the branch [arch_hostname](https://github.com/AquaQAnalytics/TorQ/tree/arch_hostname)? 

You will need to [comment out lines 32 and 33 and uncomment lines 34 and 35](https://github.com/AquaQAnalytics/TorQ/commit/8c2c99b249b6598642186decd41edacd479dd258).

Thanks,
Andrew

**Comment by jonnypress** [comment]
Closing this. Fix was merged under #323
